charts/horizon/charts/tektonci-resources/values.yaml

```diff
@@ -34,7 +34,7 @@ horizon:
 
       # config git ssh
       rm -rf /root/.ssh && ln -s /tekton/home/.ssh /root
-      echo -e 'StrictHostKeyChecking no' >> /root/.ssh/config
+      echo -e 'Host *\n    StrictHostKeyChecking no' >> /root/.ssh/config
```

Comment:
Add ' Host *\n' is for ?

Reply:
set StrictHostKeyChecking to no for all hosts
``` shell
=== ~/.ssh/config ===
Host url1.com
    HostName url1.com
    IdentityFile ~/.ssh/id_key1
Host url2.com
    HostName url2.com
    IdentityFile ~/.ssh/id_key2
Host *
    StrictHostKeyChecking no
```